docs/source/tutorials/pruning_bert_glue.py

```diff
@@ -90,8 +101,7 @@
     'wnli': ('sentence1', 'sentence2'),
 }
 
-def prepare_data(cache_dir='./data', train_batch_size=32, eval_batch_size=32,
-                    teacher_model: torch.nn.Module = None):
+def prepare_dataloads(cache_dir=data_dir, train_batch_size=32, eval_batch_size=32):
```

Comment:
this function aims to prepare dataloader, maybe prepare_dataloader is better?

Reply:
fixed